Servers/Atlas/.arc/submit.py

```diff
@@ -97,8 +104,20 @@
 
 touch initial_time
 
+SubmitDir=`pwd`
```

Comment:
Can we make this {pwd} instead? I ask because I know from experience if we rely on the script and terminal to fill in the path to working directly it can cause issues if there are brackets etc. in the party name